layer/global_context.py

```diff
@@ -45,6 +48,14 @@ def current_project_name() -> Optional[str]:
     return _GLOBAL_CONTEXT.project_name
 
 
+def set_current_account_name(name: Optional[str]) -> None:
```

Comment:
I wonder if we need to expose `set_current_account_name` and `set_current_project_name` separately? 
it feels like an opportunity to make a mistake and get into some mismatch. Could we go with `set_current_project_full_path` or is that impossible at this moment?